packages/teuchos/core/src/Teuchos_ScalarTraitsDecl.hpp

```diff
@@ -95,6 +95,8 @@ struct ScalarTraits
   typedef T halfPrecision;
   //! Typedef for double precision
   typedef T doublePrecision;
+  //! Typedef for coordinates
+  typedef T coordinateType;
```

Comment:
I don't see any specializations here for complex.  Note that PR testing may not enable complex -- you'll have to do that manually.

Reply:
Line 1135.

Reply:
Ah, OK, my bad!